framework/doc/content/source/userobjects/SidesetAroundSubdomainUpdater.md

```diff
@@ -0,0 +1,20 @@
+# SidesetAroundSubdomainUpdater
+
+!syntax description /UserObjects/SidesetAroundSubdomainUpdater
+
+## Description
```

Comment:
I don't think the double description reads that well

Reply:
You mean having a heading here? I just went with what seemed to be the standard template. You want me to remove the heading?

Reply:
I usually remove the `!syntax description` myself. That's nice when skimming over the source directory and getting the single line description of an object before clicking into the object. Once within the object though I usually find that a developer has given a much more detailed description than `params.addClassDescription` usually does such that the latter is unnecessary